src/main/resources/issues/list.jelly

```diff
@@ -12,7 +12,7 @@
 
   <div role="tabpanel" id="warningsDetails" class="tab-pane fade">
     <j:forEach var="i" items="${issues}">
-      <table class="table table-hover table-striped" id="details">
+      <table class="table table-hover table-striped details">
```

Comment:
Shouldn't this have `id="list"` (and details.jelly remains unchanged?).

Reply:
I don't think so. Giving multiple HTML Elements the same id, will break nearly every javascript/css logic.
Since this is a for-each loop, this code produces multiple tables with the same id.
As far as i know, this should not work, since every id-selector works with getElementById which returns only a single element, not a list of elements like getElementsByClassName.

Reply:
I see, I missed the for loop. Maybe the class should be called `ignore-column-sorting` then the reason for the class is obvious.

Reply:
You are right. It really should be called that way. "details" really not is a good naming to specify the behaviour. I will change this right away in the next commit.